examples/default-values-backend/tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I would clean this and convert the remainder of js files to ts, for consistency ™️